tests/zfs-tests/cmd/stride_dd.c

```diff
@@ -156,7 +156,7 @@ main(int argc, char *argv[])
 	}
 
 	if (seek > 0) {
-		if (lseek(ofd, seek * bsize, SEEK_CUR) == -1) {
+		if (lseek(ofd, (uint64_t)seek * bsize, SEEK_CUR) == -1) {
```

Comment:
`loff_t` seems like the correct type here.

Reply:
The man page for `lseek` says that the correct type is `off_t`. The type of `seek` probably should be changed to `off_t` rather than casting it to either `uint64_t` or `loff_t`. That would implicitly handle this.

Reply:
Right, that was my typo above.  I checked the man page and somehow wrote `loff_t` instead of `off_t` anyway.

Reply:
Changed to off_t